src/components/BuyButton.tsx

```diff
@@ -69,7 +70,8 @@ const BuyButton = () => {
       <Button
         role="link"
         onClick={handleClick}
-        fontSize={[1, 2, 3, 4]}
+        fontSize={[0, 1, 3]}
```

Comment:
buy button font looks a bit smaller than the rest. maybe up 1 to 2 (14px) in line with the other elements on the page